docs/solid-testing-library/api.mdx

```diff
@@ -0,0 +1,180 @@
+---
+id: api
+title: API
+sidebar_label: API
+---
+
+Due to being inspired by the [preact-testing-library][preact-docs] you can check
+its page for more information.
+
+[preact-docs]: ../preact-testing-library/intro.mdx
+
+There are several key differences, to be aware of.
+
+## Differences from the Preact Testing Library
+
+### `render`
+
+The `render` function takes in a function that returns a Solid Component, rather
+than simply the component itself.
+
+```tsx
+// With @testing-library/preact
+const results = render(<YourComponent />, options)
+```
+
+```tsx
+// With solid-testing-library
+const results = render(() => <YourComponent />, options)
+```
+
+Solid.js does _not_ re-render, it merely executes side effects triggered by
+reactive state that change the DOM, therefore there is no `rerender` method. You
+can use global signals to manipulate your test component in a way that causes it
+to update.
+
+In extension of the original API, the render function of this testing library
+supports a convenient `location` option that will set up an in-memory router
+pointing at the specified location. Since this setup is not instantaneous, you
+need to first use asynchronous queries (`findBy`) after employing it:
+
+```tsx
+it('uses params', async () => {
+  const App = () => (
+    <Routes>
+      <Route path="/ids/:id" component={() => <p>Id: {useParams()?.id}</p>} />
+      <Route path="/" component={() => <p>Start</p>} />
+    </Routes>
+  )
+  const {findByText} = render(() => <App />, {location: 'ids/1234'})
+  expect(await findByText('Id: 1234')).not.toBeFalsy()
+})
+```
+
+It uses `@solidjs/router`, so if you want to use a different router, you should
+consider the `wrapper` option instead. If you attempt to use this without having
```

Comment:
I think that this information should also be added to an FAQ but this is just an opinion and not a blocking comment :)

Reply:
I would say this information is relevant to keep on this part as it is something that impacts the usage of this render function. 🤔

Reply:
I agree on this one, that's why I said also, but we can iterate on this later if we'll find the need :)
Thanks!